boards/arm/mps3_an547/mps3_an547.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2019-2021 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mps3_an547
+name: Arm MPS3-AN547
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - zephyr
+  - xtools
+supported:
+  - gpio
+testing:
+  default: true
```

Comment:
not objecting to this, but since we do not have any 8.1 special features, maybe it would be enough, for now, that the an521 gets default: true for testing?

Reply:
Dropped it for now.